.github/workflows/publish.yml

```diff
@@ -0,0 +1,69 @@
+name: "release"
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2.1.1
+      - name: node
+        uses: actions/setup-node@v1.4.2
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org
+      - name: versioning
+        run: npm version --new-version "${GITHUB_REF:11}" --no-git-tag-version
+      - name: commit changes
+        run: |
+          git config --global user.email "robot@git.com"
+          git config --global user.name "Git Robot"
+          git add .
+          git commit -m "Update package version via git"
```

Comment:
"Sync package version with the latest git tag" ?

Reply:
Sounds better, tomorrow I fix it.